common/autoconf/libraries.m4

```diff
@@ -668,9 +668,9 @@ AC_DEFUN_ONCE([LIB_SETUP_MISC_LIBS],
   AC_MSG_CHECKING([for which zlib to use])
 
   DEFAULT_ZLIB=bundled
-  if test "x$OPENJDK_TARGET_OS" = xmacosx -o "x$OPENJDK_TARGET_OS" = xaix; then
+  if test "x$OPENJDK_TARGET_OS" = xmacosx -o "x$OPENJDK_TARGET_OS" = xaix -o "x$OPENJDK_TARGET_OS" = xlinux; then
     #
-    # On macosx and aix default is system...on others default is
+    # On macosx, aix, and linux default is system...on others default is
```

Comment:
Since you're changing this line, could you please complete it?
```
    # On macosx, aix, and linux default is system...on others default is bundled.
```

Reply:
Updated.